response.go

```diff
@@ -0,0 +1,23 @@
+package rocket
+
+type Response struct {
+	headers []Header
+	body    interface{}
+}
+
+type Header struct {
+	Key   string
+	Value string
+}
+
+func NewResponse(body interface{}) *Response {
```

Comment:
Ok, I understand why `interface{}` now. We don't have to handle it now.
But `Content-Type` is a real problem here. We can receive a JSON but response a pure text.

Reply:
I sought that content-type, maybe ok I think.

Reply:
LGTM